File: JAVA/Lab09CompareNum_DS.java
// ****684344********
// Student Name: Dilpreet Singh
// Date: 12/2/2020
// File Name: Lab09CompareNum_DS.java
// Description - Lab09 assigned 12/2/2020
// *******************
import java.util.Scanner;

public class Lab09CompareNum_DS {
        public static void main(String[] args) {
        
                Scanner scan = new Scanner(System.in);
                int integer1, integer2;
        
                System.out.print("Enter two integers: ");
                        integer1 = scan.nextInt();      
                        integer2 = scan.nextInt();
        
                System.out.println("You entered " + integer1 + " and " + integer2);
                
        
                        if ( integer1 > 0 ) {
                                System.out.println("\nThe number " + integer1 + " is positive.");
                        }
                        
                        if ( integer1 < 0 ) {
                                System.out.println("\nThe number " + integer1 + " is negative.");
                        }
                        
                        if ( integer2 > 0 ) {
                                System.out.println("The number " + integer2 + " is positive.");
                        }
                        
                        if ( integer2 < 0 ) {
                                System.out.println("The number " + integer2 + " is negative.");
                        }
                        
                        if ( integer1 % 2 == 0 ) {
                                System.out.println("\nThe number " + integer1 + " is even.");
                        }
                        
                        if ( integer1 % 2 != 0 ) {
                                System.out.println("\nThe number " + integer1 + " is odd.");
                        }
                        
                        if ( integer2 % 2 == 0 ) {
                                System.out.println("The number " + integer2 + " is even.");
                        }
                        
                        if ( integer2 % 2 != 0 ) { 
                                System.out.println("The number " + integer2 + " is odd.");
                        }
                        
                        if ( integer1 > integer2 ) {
                                System.out.println("\nThe number " + integer1 + " is larger than the number " + integer2);
                        
                                if ( integer1 % integer2 == 0) {
                                        System.out.println("The number " + integer1 + " is divisable by " + integer2);
                                        System.out.println(integer1 + " / " + integer2 + " = " + (integer1 / integer2));
                                }
                                
                                if ( integer1 % integer2 != 0) {
                                        System.out.println("The number " + integer1 + " is not divisable by " + integer2);
                                }
                        
                        }
                        
                        if ( integer2 > integer1 ) {
                                System.out.println("\nThe number " + integer2 + " is larger than the number " + integer1);
                        
                                if (integer2 % integer1 == 0 ) {
                                        System.out.println("The number " + integer2 + " is divisable by " + integer1);
                                        System.out.println(integer2 + " / " + integer1 + " = " + (integer2 / integer1));
                                }
                                
                                if (integer2 % integer1 != 0 ) {
                                        System.out.println("The number " + integer2 + " is not divisable by " + integer1);
                                }
                        
                        }
                        

                        scan.close();


                        
        
        }
}

File: PYTHON/Lab09CompareNum_DS.py
x = int(input("Enter an integer : "))

y = int(input("Enter another : "))

print("You entered are", x,"and", y, end = ".\n\n")

if x > 0:
        print("The number", x, "is positive.")
else: 
        print("The number", x, "is negative")

if y > 0:
        print("The number", y, "is positive.", end = "\n\n")
else: 
        print("The number", y, "is negative.", end = "\n\n")


if x % 2 == 0:
        print("The number", x, "is even.")
else: 
        print("The number", x, "is odd.")

if y % 2 == 0:
        print("The number", y, "is even.", end = "\n\n")
else: 
        print("The number", y, "is odd.", end = "\n\n")


if x > y:
        print("The number", x, "is larger than", y, end = ".\n")
else:
        print("The number", y, "is larger than", x, end = ".\n")


if x % y != 0 or y % x != 0:
        print("The numbers are not divisible")
else:
        if x % y == 0:
                print("The number", x, "is divisible by", y, end = ".\n")
                print(x, "/", y, "=", (x/y))

        if y % x == 0:
                print("The number", y, "is divisible by", x, end = ".\n")
                print(y, "/", x, "=", (y/x))


